LondonBritishLibrary/orient/BLorient573.xml

```diff
@@ -54,49 +429,63 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">243</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>429</height>
-                                        <width>394</width>
+                                    <dimensions type="outer" unit="in">
+                                        <height>16.875</height>
+                                        <width>15.5</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns=" 3" writtenLines="17" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>The manuscript is written in a fine hand, of the latter part of the <date>17th century</date>
                                 </desc>
-                                
+
                                 <date notAfter="1650" notAfter-custom="1700"/>
-                                
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
                         <additions>
-                            
+                        <list>
+                          <item xml:id="a1">
+                          <locus target="#3r"></locus>
+                            <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                                    <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                                <q xml:lang="gez">ጉባኤ፡ መልክ፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+                                </item>
+
+<item xml:id="a2">
+<locus target="#243r"></locus>
+<desc type="OwnershipNote">The note states that the manuscript belonged to <persName role="owner" ref="PRS13311TaklaIyasus">Takla ʾIyasus</persName>
+and his wife <persName role="owner" ref="PRS13312LahyaDengel">Lāhya Dǝngǝl</persName>. The name of Takla ʾIyasus has been erased throughout, but is still
+legible e. g. on <locus target="#231v">f. 231v</locus>.</desc>
```

Comment:
I guess the information that the name of the owner has been erased should be added to extras. BTW, you do need to write out the names, if they are marked up they will be retrieved from the corresponding file

Reply:
the type is not needed here, it  is rather a phenomena (the name was erased throughout the mansucript) than a specific text, I guess